initfiles/etc/DIR_NAME/environment.xml.in

```diff
@@ -20,6 +20,7 @@
         storageWrites="0.0038"/>
  </Hardware>
  <Software>
+  <Globals  noCommon="true" />
```

Comment:
trivial: extra space before />

Also, where has maxConnections definition gone?
The comments/discussion said the default should be picked up by dfuserver. Its default value (and any other non-binary) can be better and more easily picked up at the server, and the environment default if configured set then.

Reply:
Removed

Reply:
Again you have marked as resolved, but not released the fix.